spec/support/hmis_csv_fixtures.rb

```diff
@@ -54,6 +54,7 @@ def import_hmis_csv_fixture(
       GrdaWarehouse::Tasks::IdentifyDuplicates.new.run! unless version == '2020'
 
       GrdaWarehouse::Tasks::ProjectCleanup.new.run!
+      GrdaWarehouse::ServiceHistoryServiceMaterialized.refresh!
```

Comment:
@awisema is this intentional in this PR?

Reply:
Yes -- with an brand-new DB, the next line that calls `...ServiceHistory::Add` would blow up with PG complaining about needing to refresh the view.